gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -46,7 +46,6 @@ properties:
   additional_pathology_findings:
```

Comment:
DICT-96 says there are enums added:
Pleurodesis, Talc
Pleurodesis, NOS

I don't see them in the changes made.

Reply:
added